mean/modules/core/server/views/layout.server.view.html

```diff
@@ -36,28 +36,27 @@
 </head>
 
 <body class="ng-cloak">
-  <header ng-include="'/modules/core/client/views/header.client.view.html'" class="navbar navbar-fixed-top navbar-inverse"></header>
-  <section class="content">
-    <section class="container">
-      {% block content %}{% endblock %}
-    </section>
-  </section>
-  <footer>
-    <div class="container">
-      <p class="text-muted">Copyright 2015 WiseBuddha Enterprises LLC, Terms and Conditions, Privacy Policy, etc.</p>
-    </div>
-  </footer>
+  <div class = "container">
+    <header ng-include="'/modules/core/client/views/header.client.view.html'" class="navbar navbar-fixed-top navbar-inverse"></header>
+      <section>
+        <br><br><br>
```

Comment:
Change this to css padding please.  You can do this by putting the block content in a div tag and adding an id or class to it.